### DISCUSSION TITLE
My question is that i want to know if i use the model-repository=s3://minio/bucket/models  order , the models file will dowload to the local filesystem ? i try to enter the tritonserver docker (docker exec ) ,but i con't find the models file。

### DISCUSSION THREAD

**Discussion by Vanish1011** [triton-inference-server/server#6566] [Q&A] [answered]
My question is i want to know if i use the model-repository=s3://minio/bucket/models  order , the models file will dowload to the local filesystem ? i try to enter the tritonserver docker (docker exec ) ,but i con't find the models file。i really appricate it if anyone would reply

**Comment by Vanish1011** [comment]
@deadeyegoodwin

**Comment by nnshah1** [comment, marked as answer]
@Vanish1011 

S3 Models will be downloaded to the local filesystem in the /tmp directory (by default) - this directory can be changed via environment variable:

https://github.com/triton-inference-server/core/blob/935878a5b97d3a51b76d2e5b7fadd2f31926734c/src/filesystem/implementations/s3.h#L656

**Reply by Vanish1011** [reply]
thank you so mush for your reply👌💕
